signer/universal/universal_test.go

```diff
@@ -1 +1,21 @@
 package universal
+
+import (
+	"testing"
+)
+
+func newPKCS11Signer(t *testing.T) {
```

Comment:
I'm pretty sure go test function names have to begin with "Test", then an uppercase letter to actually be run by `go test`. Try renaming this to TestNewPKCS11Signer.